pkg/app/piped/analysisprovider/http/http.go

```diff
@@ -48,23 +48,23 @@ func NewProvider(timeout time.Duration) *Provider {
 }
 
 // Run sends an HTTP request and then evaluate whether the response is expected one.
-func (p *Provider) Run(ctx context.Context, cfg *config.AnalysisHTTP) (bool, error) {
+func (p *Provider) Run(ctx context.Context, cfg *config.AnalysisHTTP) (bool, string, error) {
```

Comment:
looks like we always return an empty string as the second return value 👀

Reply:
No wonder you wonder why. Sorry but the truth is this HTTP provider is still incubating and we're not looking to support it for a while, that's why I put temporary literal.